### PR TITLE
Feature/1.5.7/zenoh micro uri

### PR DESCRIPTION
**Pull request by MelamudMichael**

### CODE REVIEW

File: include/up-cpp/uri/builder/BuildUUri.h [hunk 1]
```diff
@@ -52,14 +52,12 @@ namespace uprotocol::uri {
          * @return 
          */
         auto setAutority(uprotocol::v1::UAuthority const &authority) -> BuildUUri & {
```
Comment (MelamudMichael) [inline include/up-cpp/uri/builder/BuildUUri.h:54]: fix typo

File: src/uri/IpAddress.cpp [hunk 2]
```diff
@@ -19,57 +19,145 @@
...
-        type_ = AddressType::IpV4;
-        length = IpAddress::IpV4AddressBytes;
-    } else if (1 == inet_pton(AF_INET6, ipString_.c_str(), &bytes)) {
-        type_ = AddressType::IpV6;
-        length = IpAddress::IpV6AddressBytes;
-    } else {
-        type_ = AddressType::Invalid;
-    }
+void uri::IpAddress::fromString() {
+    if (std::vector<uint8_t> bytes(IpV6AddressBytes, 0);
```
Comment (MelamudMichael) [inline src/uri/IpAddress.cpp:80]: can you put each of the cases in the if , in separate brackets?

File: include/up-cpp/uri/builder/BuildUUri.h (hunk 1 above)
Comment (gregmedd) [reply include/up-cpp/uri/builder/BuildUUri.h:54]: I saw that one. It touches several files. Would you mind if I make that change in a separate PR? I think the repos all need a find-and-replace for Autority->Authority.

File: src/uri/IpAddress.cpp (hunk 2 above)
Comment (gregmedd) [reply src/uri/IpAddress.cpp:80]: These if statements are in the form `if ([init-statement]; [condition])`. There is only one conditional statement within the parentheses, and typically this wouldn't require extra brackets. Are you asking me to change this to `if (([init-statement]); ([condition]))`?

**Pull request merged**